ext/opcache/Optimizer/zend_ssa.c

```diff
@@ -259,6 +259,7 @@ static void place_essa_pis(
 		switch (opline->opcode) {
 			case ZEND_JMPZ:
 			case ZEND_JMPZNZ:
+			case ZEND_JMP_NULL:
```

Comment:
This shouldn't be here, or at least the implementation is incorrect.

Reply:
I'm still learning about the details of ssa (or rather, I'd like to once I have more time). Could you elaborate on what would be correct here? Or should I just remove `ZEND_JMP_NULL` here completely?

Reply:
Thanks for fixing this :)